### DISCUSSION TITLE
Not able to access rooms / admin with local Reticulum

### DISCUSSION THREAD

**Discussion by leonardoazzi** [mozilla/hubs#3648] [General Questions & Discussion] [answered]
Hi everyone,

My team and I are trying to setup a local reticulum against our local hubs instance, in order to run the whole application. We followed the steps in readme, both in Windows (with Docker) and Ubuntu. Though it's apparentlly up and running, we are not able to enter rooms, change our user to admin or even access the admin panel. 

When I try to create and join a room, I get 'join crashed' in the console (in loop):
![image](https://user-images.githubusercontent.com/13557537/103698411-19ab5880-4f80-11eb-86c7-273e42958e9d.png)

In the iex console I get this:
![image](https://user-images.githubusercontent.com/13557537/103698577-4a8b8d80-4f80-11eb-9c65-c7540b5ec33c.png)

And when I go to the homepage (https://hubs.local:4000), I get this:
![image](https://user-images.githubusercontent.com/13557537/103698808-aa823400-4f80-11eb-8e02-2afffa016cc7.png)
![image](https://user-images.githubusercontent.com/13557537/103698841-b968e680-4f80-11eb-8347-7daca3cf41d1.png)

I successfully signed-in, but I think I can't turn the account into admin via the iex command. I get this:
![image](https://user-images.githubusercontent.com/13557537/103699462-b9b5b180-4f81-11eb-87d7-fa81aebdd1d2.png)

Could anyone help us try to solve these problems? I've ran into the discussion [#3082](https://github.com/mozilla/hubs/discussions/3082) while searching, but I think there was no definitive solution. Also found [this issue](https://github.com/mozilla/reticulum/issues/369) in reticulum rep (didn't help hahah).
Thanks a lot! :)

**Comment by robinkwilson** [comment, marked as answer]
Another discussion thread to look over: https://github.com/mozilla/hubs/discussions/3323

^^ I'd essentially reiterate what I talked about in that message: you need to run dialog locally and configure it to talk to reticulum. Right now with only Hubs and Reticulum running locally, you're failing when reticulum tries to connect to the hubs dev dialog server (understandably, you're missing credentials). 

Unfortunately, I've yet to see a local setup running all three reticulum/hubs/dialog working without the developer credentials. You're totally welcome to try and get this working, I can't guarantee the amount of help we can give you since we're a small team.

More relevant links:
https://github.com/mozilla/hubs/issues/2565

**Reply by leonardoazzi** [reply]
Thanks, @robinkwilson. 

I was not so aware of dialog and the developer credentials, but later I found some comments and recent discussions about it.

We are testing the system and, if we have success, we will be sharing with you right away.

**Reply by samkohli** [reply]
Once you go into dialog. Make sure to run:

npx browserslist@latest --update-db

or the error will still appear

**Comment by agriggs2** [comment]
I believe I found the issue with running the whole hubs stack locally. 
`Missing file index.html. Please try again.`
from what I can tell the admin dashboard is trying to access https://hubs.local:4000/api/v1/app_configs but no matter what it returns {"error":"unauthenticated"} so it looks like you have to have an account prior to running  https://hubs.local:4000/?skipadmin or going to admin dashboard. 

any way to get around this?
